WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -914,6 +922,16 @@ private function validate_prefixes() {
 				continue;
 			}
 
+			if ( strlen( $prefix ) < self::MIN_PREFIX_LENGTH ) {
```

Comment:
This should be `mb_strlen()` since multibyte characters could be used as prefixes - see https://3v4l.org/qTg7h as a quick example.